.travis.yml

```diff
@@ -1,12 +1,19 @@
+dist: xenial
+sudo: false
+
 language: python
 python:
   - "2.7"
-  - "3.4"
   - "3.5"
   - "3.6"
-  - "pypy-5.4.1"
+  - "3.7"
+  - "pypy"
```

Comment:
Can you try `pypy2.7` ? If it still doesn't work, this one should work I think: `pypy2.7-6.0`

Reply:
https://travis-ci.community/t/confusing-pypy-versions/2505